tool/lib/webqa-manual-annotations.js

```diff
@@ -369,22 +372,26 @@ const MANUAL_PROPERTY_CANONICAL_OVERRIDE = {
     actor: {
         base: ['actor', 'actress'],
         property: ['#'],
-        passive_verb: ['played by', 'acted by']
+        passive_verb: ['played by', 'acted by'],
+        reverse_verb: ['acted', 'acted in']
```

Comment:
What about "was in" and "is in" (as in "who was in titanic?") Is that something you can get with "preposition"? Is that a reversed verb?

Reply:
Added (it should be reverse verb. preposition would be "in Leonardo DiCapiro" not "in Titanic")